src/Bounds.cpp

```diff
@@ -794,6 +794,22 @@ class Bounds : public IRVisitor {
     }
 
     void visit(const Call *op) {
+        // Using the strict_float feature flag wraps a strict_float()
+        // call around every Expr that is of type float, so it's easy
+        // to get nestings that are many levels deep; the bounds of this
+        // call are *always* exactly that of its first argument, so short
+        // circuit it here before checking for const_args. This is important
+        // because evaluating const_args for such a deeply nested case
+        // essentially becomes O(n^2) doing work that is unnecessary, making
+        // otherwise simple pipelines take several minutes to compile.
+        //
+        // TODO: are any other intrinsics worth including here as well?
```

Comment:
I guess any call that we expect to be stacked > O(1) times... I can't immediately think of another.

Reply:
If there are any where we know const_args is always false or irrelevant.